src/GraphQL.Upload.AspNetCore/ApplicationBuilderExtensions.cs

```diff
@@ -40,7 +40,7 @@ public static IApplicationBuilder UseGraphQLUpload<TSchema>(this IApplicationBui
             if (options is null)
                 throw new ArgumentNullException(nameof(options));
 
-            return builder.UseMiddleware<GraphQLUploadMiddleware<TSchema>>(options);
```

Comment:
graphQL path is now passed into GraphQLUploadMiddleware (instead of unused previously)

Reply:
Thanks for noticing!